libfwupdplugin/fu-common.c

```diff
@@ -1923,3 +1929,37 @@ fu_byte_array_append_uint32 (GByteArray *array, guint32 data, FuEndianType endia
 	fu_common_write_uint32 (buf, data, endian);
 	g_byte_array_append (array, buf, sizeof(buf));
 }
+
+/**
+ * fu_common_kernel_locked_down:
+ *
+ * Determines if kernel lockdown in effect
+ *
+ * Since: 1.3.8
+ **/
+gboolean
+fu_common_kernel_locked_down (void)
```

Comment:
could we add some tests for this please?

Reply:
Added